File: Programming Basics/Programming Basics - C#/Exams/Programming Basics Exam - 28 August 2016/Exam - 28 August 2016/4. Hospital/Hospital.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _4.Hospital
{
    class Hospital
    {
        static void Main(string[] args)
        {
            int days = int.Parse(Console.ReadLine());
            int doctors = 7;
            double treated = 0;
            double untreated = 0;

            for (int rowPatiesnts = 1; rowPatiesnts <= days; rowPatiesnts++)
            {
                double patients = double.Parse(Console.ReadLine());

                if (rowPatiesnts % 3 == 0)
                {
                    if (untreated > treated)
                    {
                        doctors++;
                    }
                }

                if (patients <= doctors)
                {
                    treated = treated + patients;
                }

                else
                {
                    treated = treated + doctors;
                    untreated = untreated + (patients - doctors);
                }
            }
            Console.WriteLine("Treated patients: {0}.", treated);
            Console.WriteLine("Untreated patients: {0}.", untreated);
        }
    }
}

File: Programming Basics/Programming Basics - JavaScript/Exams/Programming Basics Exam - 28 August 2016/4. Hospital/Hospital.js
function hospital(args) {
    let days = Number(args[0]);
    let doctors = 7;
    let treated = 0;
    let untreated = 0;

    for (let rowPatiesnts = 1; rowPatiesnts <= days; rowPatiesnts++)
    {
        let patients = Number(args[1]);

        if (rowPatiesnts % 3 == 0) {
            if (untreated > treated) {
                doctors++;
            }
        }

        if (patients <= doctors) {
            treated = treated + patients;
        }

        else {
            treated = treated + doctors;
            untreated = untreated + (patients - doctors);
        }
    }
    console.log(`Treated patients: ${treated}.`);
    console.log(`Untreated patients: ${untreated}.`);
}

/* Uncomment to test:
hospital(["3", "7", "7", "7"]);
    */